### ISSUE TITLE
Candidate: kayabaNerve for MAGIC Monero Fund

### ISSUE THREAD

**Issue by kayabaNerve** [MAGICGrants/Monero-Fund#7]
- [x] I have read and understand the [MAGIC Monero Fund documents](https://magicgrants.org/funds/monero/).
- [x] I have read and understand the relevant [MAGIC Grants policies](https://magicgrants.org/about/documentation/).
- [x] I understand that should I be in a position to receive funds from MAGIC Grants for any reason, I will need to provide identifying information to the organization privately. Election to a committee does not guarantee compensation even if compensation is made available, since MAGIC Grants has legal limitations on how it can disburse funds.
- [x] I understand that an active committee is the number one factor in a Fund's success.
- [x] I am at least 18 years old.

I'm Luke Parker, more frequently known as Kayaba/kayabaNerve. I have contributed to the ecosystem in a few different ways, notably with [ASMR](https://github.com/MerosCrypto/asmr), this first implementation of the Monero atomic swap protocol. I'm a developer at heart with years of experience working with cryptocurrencies, on the protocol, node, and ecosystem levels, and I believe that puts me in a well established position in order to evaluate grants.

I'm specifically running for the Monero fund because I believe in financial privacy. Monero is the top privacy coin and there's a lot of reasons for why. What I most admire about Monero is how it's been a community effort, without corporations behind it, based on people and code. Unfortunately, recent times have shown how the CCS, while incredible, has its short comings. I believe establishing a 501(c)3 option to fund Monero development is critical in order to maintain researcher and developer relationships, and MAGIC is perfectly positioned for that.

Beyond making sure we are able to sustain relationships without so many concerns about funding, which has caused us to lose people in the past, I want to focus on accessibility. I believe every cryptocurrency out there will need to learn to cooperate on some level, or they will die alone. The existence of projects based solely around interoperability proves that for me. For Monero, that means furthering atomic swaps to the point they're mass accessible and mass used, not to mention integrations with other protocols (various DEXs). I also want to encourage development which otherwise extends Monero, such as payment channels which have had multiple recent theoretical developments.

I'm on IRC as kayabaNerve, along with Matrix which I check less often, and my email is lukeparker5132@gmail.com. I am happy to answer questions and wish this endeavor the best, as I believe it is critical for Monero. I want to be able to do my best to further it and ensure Monero's prosperous future.

**Comment by carrington1859** [comment]
Switch to a secure email service and then you have my vote.

**Comment by kayabaNerve** [comment]
I have considered a Protonmail/Tutanota address before, yet have found I am obligated to use Google services in my life for various reasons. While I would still offer a Protonmail contact in order for other people to reach out securely, I instead maintain a KeyBase and Matrix account which I can be contacted on, both offering that functionality (or, for people I trust more, a Signal though that of course doxxes my phone number making it not something I care to publicize freely). This is due to my personal preference for IMs over emails which continues to show its lack of functionality and practicality as the years go by. 

If you do have a specific reason to need a secure email service, and not just a secure messaging service in general, please feel free to let me know so I can re-evaluate my position. It would increase the amount of messages I handle which are encrypted (as all emails would have somewhat better security, and all emails which interact with other Protonmail accounts would be significantly better), yet I have yet to find those factors significant enough when the vast majority of emails I receive already are handled by Google in some form (and therefore doxxed to Google, though I do see how by having a Google account I'm contributing to that problem) and I've had this email for long enough that migrating would be a pain.

TL;DR I do have secure communication options available. I haven't personally felt it would notably increase my security to migrate. Migrating is a pain.
